libvast/test/synopsis.cpp

```diff
@@ -80,11 +79,14 @@ FIXTURE_SCOPE(synopsis_tests, fixtures::deterministic_actor_system)
 
 TEST(serialization) {
   factory<synopsis>::initialize();
+  // auto x = synopsis_ptr{};
+  // CHECK_EQUAL(roundtrip(x), x);
+
```

Comment:
This is probably not supposed to be here, still.

Reply:
They really need not be here. Removed them.